Assets/Scripts/EnemySpawner.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemySpawner : MonoBehaviour
+{
+    [SerializeField] private List<GameObject> planes;
+
+    [SerializeField] public bool planeSpawned;
+
+    [SerializeField] private float spawnCooldown;
+
+    public GameObject currentPlane;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        SetSpawnCooldown();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        spawnCooldown -= Time.deltaTime;
+
+        if (spawnCooldown <= 0 && !planeSpawned)
+        {
+            var randomPlane = Random.Range(0, planes.Count);
+            Instantiate(planes[randomPlane], transform.position, Quaternion.identity);
+            planeSpawned = true;
+        }
+
+        currentPlane = GameObject.FindWithTag("Enemy");
```

Comment:
```suggestion
      var currentPlane = GameObject.FindWithTag("Enemy");
```
dan kan deze bovin weg, is niet nodig